lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -71,7 +71,7 @@ def self.configure(rails_config)
           tracer.default_service = Datadog.configuration[:rails][:default_service]
 
           Datadog.configuration[:rack][:tracer] = tracer
-          Datadog.configuration[:rack][:default_service] = Datadog.configuration[:rails][:default_service]
+          Datadog.configuration[:rack][:service_name] = Datadog.configuration[:rails][:default_service]
```

Comment:
@p-lambert can't we call it `service_name` like others? since we're removing a lot of legacy options, I'd say to make names consistent, so `service_name` instead `default_service` like for other integrations.

Reply:
I did that change for Sinatra that will land in the 0.10.0 release so we have only one breaking change for Sinatra. Thoughts?

Reply:
Ref PR: https://github.com/DataDog/dd-trace-rb/pull/260